### PR TITLE
capability to prevent form submit (testcafe #3560)

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

File: src/client/sandbox/node/element.ts [hunk 1]
```diff
@@ -502,9 +502,15 @@ export default class ElementSandbox extends SandboxBase {
 
             formSubmit () {
                 sandbox._ensureTargetContainsExistingBrowsingContext(this);
-                sandbox.emit(sandbox.BEFORE_FORM_SUBMIT_EVENT, { form: this });
 
-                return nativeMethods.formSubmit.apply(this, arguments);
+                const args = { form: this, preventSubmit: false };
```
Comment (AndreyBelym) [inline src/client/sandbox/node/element.ts]: Could you please add a `// HACK: ` comment with a short description and a link to the corresponding TestCafe issue?

**Comment by AlexKamaev** [comment]
@LavrovArtem

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot  retest

**Pull request merged**